lib/common/constants.js

```diff
@@ -326,7 +328,9 @@ function constantsFactory (path) {
             Pdu: 'pdu',
             Mgmt: 'mgmt',
             Enclosure: 'enclosure',
-            Rack: 'rack'
+            Rack: 'rack',
+            Storage: 'storage',
```

Comment:
not a comment, just be curious why 'Storage' rendered color is different with others :)